### DISCUSSION TITLE
Display result type "URL" as a hyperlink to an external web page

### DISCUSSION THREAD

**Discussion by taliamath** [microsoft-search/pnp-modern-search#3279] [Q&A]
Hello community,

I am trying to display a hyperlinked URL for a "Link" content type in a SharePoint PnP custom search interface. I have followed suggestions found on this board, and the one thing that almost worked is creating a custom field linking ows_URL (my custom URL column) to a refinablestring managed property, calling it as a selected property, adding it to Slots, and putting this code in the template: {{slot item @root.slots.URL}}. If I try to add any variations of HTML that it seems like should convert it to a hyperlink, following various suggestions, it does not work. What I see now in the interface is this: `[InternetShortcut] URL=https://linktosite.html.` If anyone has any suggestions, I would be very grateful. Thank you.

**Comment by wobba** [comment]
Take a look at the `getUrlField` helper listed on https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#handlebars-helpers which works if your URL field is a site column with an auto created managed property.
